folder.rb

```diff
@@ -0,0 +1,62 @@
+require 'fileutils'
+
+class Folder
+
+  def self.check_folder(folder)
+
+    return false if !File.directory?(folder)
+
+    folders = Dir.entries(folder)
+
+    if check_important(folders).inspect == 'true'
+      return true
+    end
+
+    folders.each do |f|
+      next if f.include?(".")
+      if File.directory?("#{folder}/#{f}")
+        subfolder= folder + "/" + f
+        puts "Found subfolder: " + subfolder
+        return true if check_folder(subfolder).inspect == 'true'
+      end
+    end
+  end
+
+  def self.check_important(folders)
+    #puts "Checking important files in folder: " + folders.to_s
+    if folders.include?".important"
```

Comment:
```suggestion
    return false unless folders.include?(".important")
    
    puts "Found '.important' file"
    true
```
Can you use guard clause here